### DISCUSSION TITLE
API V2  erro

### DISCUSSION THREAD

**Discussion by Bluettiniko** [tweepy/tweepy#2090] [Questions]
SOS, pls A hero helps me to figure it out what's going on here. I have learned some coding during school, since Twitter API V2  i have update my coding. but it keep error
[Untitled2-Copy1.pdf](https://github.com/tweepy/tweepy/files/11221379/Untitled2-Copy1.pdf)

**Comment by Harmon758** [comment]
Double check that your credentials are correct. Regenerate them if necessary.

In the future, you need to provide the full traceback and relevant code, not as a PDF, as well as a more specific description of your issue.
